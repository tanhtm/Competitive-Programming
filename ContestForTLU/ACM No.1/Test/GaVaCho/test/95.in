397778462 899941598
