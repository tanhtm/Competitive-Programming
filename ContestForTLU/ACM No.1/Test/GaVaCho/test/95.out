345586125 52192337
